packages/nimble-components/src/select/styles.ts

```diff
@@ -116,6 +118,18 @@ export const styles = css`
         overflow: auto;
     }
 
+    ::slotted([role='option'][active-option]) {
+        background-color: ${fillSelectedColor};
+    }
+
+    ::slotted([role='option']:not([active-option])) {
+        background: none;
+    }
```

Comment:
Can we avoid the :not selector by switching the order and changing to color?
Main goal is to [avoid not selectors and rely on cascade](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#prefer-cascade-for-state-changes)

```suggestion
    ::slotted([role='option']) {
        background-color: transparent;
    }
    ::slotted([role='option'][active-option]) {
        background-color: ${fillSelectedColor};
    }
```